src/better-bem.ts

```diff
@@ -15,22 +31,22 @@ const GLUE_PROP = '-';
 /**
  * cleanClassNamesArray
  *
- * @param {string|array|Object} input               classnames which should be cleaned
- * @param {Boolean}             useKeyValuePairs    if true handle prop-value mod classnames
+ * @param input               classnames which should be cleaned
+ * @param useKeyValuePairs    if true handle prop-value mod classnames
  *
- * @return {array} clean array clean classnames
+ * @return clean array clean classnames
  */
-const cleanClassNamesArray = (input = [], useKeyValuePairs = false, keyValueGlue) => (
+const cleanClassNamesArray = (input: ClassNames = [], useKeyValuePairs = false, keyValueGlue = ''): string[] => (
```

Comment:
I had to add a default value to `keyValueGlue` since TS doesn't allow mandatory arguments not to be filled.